src/bint64.ml

```diff
@@ -1,55 +1,73 @@
 (* This file is free software, part of containers. See file "license" for more details. *)
 
-include Int64
+type t = int64
 
-let (+) = add
+let (+) = Int64.add
 
-let (-) = sub
+let (-) = Int64.sub
 
-let (~-) = neg
+let (~-) = Int64.neg
 
-let ( * ) = mul
+let ( * ) = Int64.mul
 
-let (/) = div
+let (/) = Int64.div
 
-let (mod) = rem
+let (mod) = Int64.rem
 
-let (land) = logand
+let (land) = Int64.logand
 
-let (lor) = logor
+let (lor) = Int64.logor
 
-let (lxor) = logxor
+let (lxor) = Int64.logxor
 
-let lnot = lognot
+let lnot = Int64.lognot
 
-let (lsl) = shift_left
+let (lsl) = Int64.shift_left
 
-let (lsr) = shift_right_logical
+let (lsr) = Int64.shift_right_logical
 
-let (asr) = shift_right
+let (asr) = Int64.shift_right
 
-let equal (x:t) y = x=y
+let equal (x:t) y = Pervasives.compare x y = 0
 
-let hash x = Pervasives.abs (to_int x)
+let hash x = Pervasives.abs (Int64.to_int x)
+
+let maxInt = Int64.max_int
+
+let minInt = Int64.min_int
+
+let abs = Int64.abs
 
 (** {2 Conversion} *)
 
-let of_int_exn = of_int
+let ofIntExn = Int64.of_int
```

Comment:
Should be `fromIntOrRaise`. There's quite a few other `of*` functions that should be `from*`, and a few other instances of `*Exn` that should be `*OrRaise`.